README.md

```diff
@@ -40,7 +40,29 @@ For more information, see [What is Internet Identity?](https://internetcomputer.
 
 This section gives an overview of Internet Identity's architecture, instructions on how to build the Wasm module (canister), and finally pointers for integrating Internet Identity in your own applications.
 
-### Architecture overview
+### Local Replica
+
+Use the Internet Identity canister in your local dfx project by adding the following code snippet to your `dfx.json` file:
```

Comment:
Could you also leave a sentence pointing to `dfx deps` which is also supported by II? Just as an alternative, not to replace the snippet.

Reply:
Done! I added a step-by-step explanation, although it could also make sense just to point to the `dfx deps` documentation if that's your preference.